### ISSUE TITLE
loss doesn't decrease and keep on 40 in my dataset

### ISSUE THREAD

**Issue by guapizyq** [Adamdad/keras-YOLOv3-mobilenet#24]
I train with my own dataset with the default script
my dataset has three types of objection with 40000 train sample and 6000 validation sample
the batch_size is 4, my loss decrease to 49 in the first epoch, but loss keep on 40 in the 13th epoch,  the loss doesn't decrease anymore. 
how should i change my learning rate and others parameter

**Comment by guapizyq** [comment]
it would be appreciated if you give some advise on  this issue

**Comment by Adamdad** [comment]
Can you detect anything in the testset？if not，what is your learning rate,lr decay pacience?

**Comment by guapizyq** [comment]
I am sorry, it seems to my mistakes.
the first training is to get a stable loss
the second training without frozen layers is to get a lower loss?

**Comment by Adamdad** [comment]
correct

**Comment by guapizyq** [comment]
I got a lower loss than 40, but it still is 38
the result in my testdset looks ok, i do not konw how to decrease the loss

**Comment by zhangyufei1995** [comment]
@guapizyq @Adamdad  I am very happy to discuss with you. What I want to ask is, 1. How much is the epoch setting of the red arrow here? 2, how much is the initial_epoch setting of the two black arrows? I look forward to your answer.
![QQ图片20190531173008](https://user-images.githubusercontent.com/45422726/58696270-d105e380-83c9-11e9-9e52-4b406c131b73.png)

**Comment by zhangyufei1995** [comment]
And why is it divided into step-by-step training?the first training? the second training ?What is their role?

**Comment by Adamdad** [comment]
The first training part is for finetuning a model quickly. It freezes most layers, only to train on the last few layers. We can get an acceptable model for detection in a short period of time

The second training part is for getting a complete model. All the layers can be trained through this process.

Under most occasions, I only use the second part. Epoch under is not important here.

**Comment by gzz1529657064** [comment]
> The first training part is for finetuning a model quickly. It freezes most layers, only to train on the last few layers. We can get an acceptable model for detection in a short period of time
> 
> The second training part is for getting a complete model. All the layers can be trained through this process.
> 
> Under most occasions, I only use the second part. Epoch under is not important here.

After model training, I have a model with size of 277M. It is bigger than YOLO-v3,Why?
Doesn't MobileNet reduce model parameters?
This is my training strategy in my dateset.
1. Unfreeze all of the layers
2. learning_rate = 0.001
3. load_pretrained=False
4. batch_size = 16
